File: docs/src/testers/functions/invoke_sdk.py
# @sniptest filename=invoke_sdk.py
import os

from notte_sdk import NotteClient

client = NotteClient()
function = client.Function(function_id=os.environ["NOTTE_FUNCTION_ID"])

# Via SDK
result = function.run(url="https://example.com", search_query="laptop")

print(result.result)

File: docs/src/testers/functions/invoke_sdk.ts
// @sniptest filename=invoke_sdk.ts
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();
const fn = client.NotteFunction({ function_id: process.env.NOTTE_FUNCTION_ID! });

const result = await fn.run({ url: 'https://example.com', search_query: 'laptop' });

console.log(result.result);
